src/Metadata/Resource/Factory/OperationResourceMetadataFactory.php

```diff
@@ -123,6 +123,8 @@ private function normalize(bool $collection, ResourceMetadata $resourceMetadata,
                 $supported ? $operation['method'] = $upperOperationName : $operation['route_name'] = $operationName;
             }
 
+            $operation['method'] = strtoupper($operation['method']);
```

Comment:
This does not seem correct to me. `$operation['method']` might not be set.

Anyway, this part of the code could probably use some refactoring. `$upperOperationName` is... weird. :see_no_evil: